NICE.Identity.Management/ClientApp/src/views/Organisation/Organisation.tsx

```diff
@@ -0,0 +1,166 @@
+import { ToFormattedDateString } from "../../helpers/dateHelpers";
+
+type TParams = { id: string };
+
+type LocationState = {
+	hasBeenEdited: boolean;
+};
+
+type OrganisationProps = Record<string, unknown> &
+	RouteComponentProps<TParams, StaticContext, LocationState>;
+
+type OrganisationState = {
+	organisation: OrganisationType;
+	users: Array<UserType>;
+	error?: Error;
+	redirect: boolean;
+	isLoading: boolean;
+};
+
+export class Organisation extends Component<OrganisationProps, OrganisationState> {
+		constructor(props: OrganisationProps) {
+			super(props);
+	
+			this.state = {
+				organisation: {} as OrganisationType,
+				users: [],
+				redirect: false,
+				isLoading: true,
+			};
+		}
+
+		async componentDidMount(): Promise<void> {
+			this.setState({ isLoading: true });
+	
+			const organisation = await fetchData(Endpoints.organisation(this.props.match.params.id));
+			const users = await fetchData(Endpoints.usersByOrganisation(this.props.match.params.id));
+
+			if (isDataError(users)) {
+				this.setState({ error: users });
+			}
+
+			this.setState({ organisation, users, isLoading: false });
+		}
+
+	render(): JSX.Element {
+		const { organisation, users, error, isLoading } = this.state;
+
+		return (
+		<>
+			
+			<Breadcrumbs>
+				<Breadcrumb to="/overview" elementType={Link}>
+					Administration
+				</Breadcrumb>
+				<Breadcrumb>Organisation</Breadcrumb>
+				<Breadcrumb>{organisation.name}</Breadcrumb>
+			</Breadcrumbs>
+
+			{!error ? (
+				<>
+					<PageHeader heading={organisation.name} className="page-header mb--d" />
+							<>
+								<Button
+									data-qa-sel="edit-organisation-button"
+									variant="cta"
+									to={`/organisations/${this.props.match.params.id}/edit`}
+									elementType={Link}
+									disabled={isLoading}
+								>
+									{isLoading ? "Loading..." : "Edit organisation"}
+								</Button>
+								<Button
+									data-qa-sel="edit-organisations-users-button"
+									to={`/users-organisations/${this.props.match.params.id}/edit`}
+									elementType={Link}
+									disabled={isLoading}
+								>
+									Edit users
+								</Button>
+							</>
+					<Grid>
+						<GridItem cols={12} md={9} aria-busy={isLoading}>
+									<div className={`${styles.summaryList} pv--c`}>
+										<span className={styles.summaryListLabel}>
+											Date added
+										</span>
+										<span
+											className={styles.summaryListDetail}
+											data-qa-sel="dateAdded-organisation"
+										>
+											{ToFormattedDateString(organisation.dateAdded)}
+										</span>
+									</div>
+						</GridItem>
+
+						<GridItem cols={12} md={9} aria-busy={isLoading}>
+						{isLoading ? (
+								<p>Loading...</p>
+							) : users.length ? (
+									<div className={`${styles.summaryList} pv--c`}>
+										<span className={styles.summaryListLabel}>
+											Users
+										</span>
+										<span className={styles.summaryListDetail}>
```

Comment:
I'll stop copying the styles comment, but basically if we don't have a Organisation.module.scss file, we can remove all the references to the styles.summary class stuff or alternatively we need to create a module file that has the same copied styles from Users.module.scss.